client/my-sites/domains/domain-management/settings/cards/transferred-domain-details.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useMyDomainInputMode } from 'calypso/components/domains/connect-domain-step/constants';
+import { transferStatus } from 'calypso/lib/domains/constants';
+import { INCOMING_DOMAIN_TRANSFER_STATUSES } from 'calypso/lib/url/support';
+import { domainUseMyDomain } from 'calypso/my-sites/domains/paths';
+import type { DetailsCardProps } from './types';
+
+import './style.scss';
+
+const TransferredDomainDetails = ( {
+	domain,
+	isLoadingPurchase,
+	selectedSite,
+}: DetailsCardProps ): JSX.Element => {
+	const translate = useTranslate();
+
+	const renderStartTransferButton = () => {
+		if ( ! domain.currentUserIsOwner || transferStatus.PENDING_START !== domain.transferStatus ) {
+			return null;
+		}
+
+		return (
+			<Button
+				primary
+				href={ domainUseMyDomain(
+					selectedSite.slug,
+					domain.name,
+					useMyDomainInputMode.startPendingTransfer
+				) }
+				disabled={ isLoadingPurchase }
+			>
+				{ translate( 'Start transfer' ) }
+			</Button>
+		);
+	};
+
+	const getDescriptionText = () => {
+		const { currentUserIsOwner, name, owner } = domain;
+		if ( transferStatus.PENDING_START === domain.transferStatus ) {
+			return currentUserIsOwner
+				? translate(
+						'We need you to complete a couple of steps before we can transfer %(domain)s from your ' +
```

Comment:
I would just wrap %(domain)s with `strong`, in order to standardize with other messages.